testsuite/ext-tools/maintenance_json_generator.py

```diff
@@ -25,10 +25,12 @@
                         "/SUSE_Updates_SLE-Product-SLES_15-SP2-LTSS_x86_64/"],
     "sle15sp3_client": ["/SUSE_Updates_SLE-Manager-Tools_15_x86_64/",
                         "/SUSE_Updates_SLE-Module-Basesystem_15-SP3_x86_64/",
-                        "/SUSE_Updates_SLE-Module-Server-Applications_15-SP3_x86_64/"],
+                        "/SUSE_Updates_SLE-Module-Server-Applications_15-SP3_x86_64/",
+                        "/SUSE_Updates_SLE-Product-SLES_15-SP3-LTSS_x86_64/"],
```

Comment:
this is wrong. we are not getting anymore  updates  delivered to 15-sp3-ltss

Reply:
this is for the clients, the customers can still have LTSS module enabled in minion and client, no?

Reply:
clarified in slack, this line is fine, customers can have LTSS in SLES minions and clients